tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package configure
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	defaultRPCTxFeeCap = 100
+	newRPCTxFeeCap1    = 101
+	newRPCTxFeeCap2    = 102
+	acpSupport1        = 111
+	acpSupport2        = 112
+	node1CertPath      = "tests/e2e/assets/node1/staker.crt"
+	node2CertPath      = "tests/e2e/assets/node2/staker.crt"
+	node1TLSPath       = "tests/e2e/assets/node1/staker.key"
+	node2TLSPath       = "tests/e2e/assets/node2/staker.key"
+	node1BLSPath       = "tests/e2e/assets/node1/signer.key"
+	node2BLSPath       = "tests/e2e/assets/node2/signer.key"
+	node1ID            = "NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu"
+	node2ID            = "NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5"
+)
+
+func checkBlockchainConfig(
+	nodesInfo map[string]utils.NodeInfo,
+	blockchainID string,
+	expectedRPCTxFeeCap int,
+	nodesRPCTxFeeCap map[string]int,
+) {
+	for nodeID, nodeInfo := range nodesInfo {
+		logFile := path.Join(nodeInfo.LogDir, blockchainID+".log")
+		fileBytes, err := os.ReadFile(logFile)
+		gomega.Expect(err).Should(gomega.BeNil())
+		if nodesRPCTxFeeCap != nil {
+			var ok bool
+			expectedRPCTxFeeCap, ok = nodesRPCTxFeeCap[nodeID]
+			gomega.Expect(ok).Should(gomega.BeTrue())
+		}
+		gomega.Expect(fileBytes).Should(gomega.ContainSubstring("RPCTxFeeCap:%d", expectedRPCTxFeeCap))
+		for _, rpcTxFeeCap := range []int{defaultRPCTxFeeCap, newRPCTxFeeCap1, newRPCTxFeeCap2} {
+			if rpcTxFeeCap != expectedRPCTxFeeCap {
+				gomega.Expect(fileBytes).ShouldNot(gomega.ContainSubstring("RPCTxFeeCap:%d", rpcTxFeeCap))
+			}
+		}
+	}
+}
+
+func getBlockchainConfig(rpcTxFeeCap int) string {
```

Comment:
This should go in some utils I think

Reply:
I believe it is too much specific for this test. will move just into a utils.go inside the same package